roles/edpm_tripleo_cleanup/meta/argument_specs.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+argument_specs:
+  # roles/edpm_tripleo_cleanup/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_tripleo_cleanup role.
+    options:
+      edpm_old_tripleo_services:
+        type: "list"
+        description: |
+          List of tripleo services to be disabled and stopped.
+          If the list is empty, all services containing string 'tripleo'
+          in their names will be disabled, stopped and masked.
+        default: []
+      edpm_remove_tripleo_unit_files:
+        type: "bool"
+        description: |
+          Remove unit files after disabling services.
```

Comment:
in two minds about this one. where do we draw the cleanup line? It is one thing to stop the service it is another to cleanup files/packages etc. (yes I realise you're not removing packages here).

Reply:
At this point I don't really care much either way. The cleanup procedure can be adjusted, or left with defaults. One way or another, given the number of files involving tripleo, we are going to miss something. After all, we have been adopting without cleanup up until now. 

What is critical, is that no tripleo processes are running, or can spontaneously start up after adoption. As that could cause damage.

Reply:
I think when we start doing operations on the data plane we are already beyond "point of no return" in terms of rollback so i don't think we'd be reinstating the old services. But i think it's nice that there is a variable to control the removal, thanks.